vendor/bat-native-ledger/BUILD.gn

```diff
@@ -102,6 +106,7 @@ source_set("ledger") {
   ]
 
   deps = [
+    "//base",
```

Comment:
@bridiver can we add this?

Reply:
yes, we've discussed using `base` and other "utility" classes from chromium inside ledger. For now we should only be using them internally and not exposing anything in the public API